utils/xray/xrayutils.go

```diff
@@ -7,19 +7,23 @@ import (
 const MaxUniqueAppearances = 10
 
 type DepTreeNode struct {
-	Classifier *string   `json:"classifier"`
-	Types      *[]string `json:"types"`
-	Children   []string  `json:"children"`
+	Classifier     *string   `json:"classifier"`
+	Types          *[]string `json:"types"`
+	Children       []string  `json:"children"`
+	Unresolved     bool      `json:"unresolved,omitempty"`
+	Configurations *[]string `json:"configurations,omitempty"`
```

Comment:
Is there a use for them here other than the logs? do we want to skip unresolved?
please make sure Xray can scan those dependencies by adding a test scan on a similar project and making sure Xray can handle those deps